### PR TITLE
Feat/client insert product req

### PR DESCRIPTION
**Pull request by SkimLiu**
### Description of the PR

Add sprint doc

----------
### Additional changes

None

### CODE REVIEW

File: restrospective/sprint1.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+#### Code review:
+
+Not specifically estimated, more included in the other tasks, a good estimate would be around half an hour per person, so 2.5 hrs in total.
+
+#### Technical Debt management:
+
+Not planned for this second sprint other than reviewing SonarQube and taking that into the next sprint.
+
+​	Ratings: No data
```
Comment (shantifabri) [inline restrospective/sprint1.md:63]: This is a title, the data is literally below.

File: restrospective/sprint1.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+#### Detailed statistics
+
+| Stroy                                     | #Tasks | Points | Hours est. | Hour Actual |
+| ----------------------------------------- | ------ | ------ | ---------- | ----------- |
+| Learn Flask + Setup DB + Porject Template | 4      |        | 16         | 19          |
+| Enter Client Order                        | 2      | 8      | 6          | 5           |
+| Enter New Client                          | 2      | 10     | 8          | 6           |
+| Browse Products                           | 2      | 21     | 8          | 6           |
+| Registration                              | 3      | 8      | 2          | 2           |
```
Comment (shantifabri) [inline restrospective/sprint1.md:33]: Profesor asked if we could add the story id's

File: restrospective/sprint1.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+- What caused your errors in estimation (if any)?
+
+The wrong assessments for some tasks, it causes us to spend more time on it directly.
+
+- What lessons did you learn (both positive and negative) in this sprint?
+
+P1: I learned how to audit python code and try to fix them, it improves my ability in security. And this sprint also told me I should talk with our team more frequently.
+
+P2: We have learnt that we need to be more collaborative if we want to achieve great results. Poor collaboration leads to discrete results and that is not what we want.
```
Comment (shantifabri) [inline restrospective/sprint1.md:87]: Don't use individual perspectives, write as a group

**Review by shantifabri — CHANGES_REQUESTED** [review]

File: restrospective/sprint1.md (hunk 3 above)
Comment (SkimLiu) [reply restrospective/sprint1.md:87]: And give more words on teamwork, not including manager or other

**Pull request closed without merging**